Unable to install ocamlfind

Opam installed succesfully on a Mac Air, El Capitan (10.11)
Ocaml and camlp5 installed succesfully.
Failed to install ocamlfinf!!!

```
MacAir-H:~ Henri$ opam install ocamlfind
The following actions will be performed:
  ∗  install conf-pkg-config 1.0              [required by conf-ncurses]
  ∗  install conf-ncurses    1                [required by ocamlfind]
  ∗  install ocamlfind       1.6.1
===== ∗  3 =====
Do you want to continue ? [Y/n] y

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ocamlfind] Archive in cache

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] The compilation of conf-pkg-config failed at "pkg-config --help".

#=== ERROR while installing conf-pkg-config.1.0 ===============================#
# opam-version 1.2.2
# os           darwin
# command      pkg-config --help
# path         /Users/Henri/ocamlbrew/ocaml-4.02.3/.opam/4.02.1/build/conf-pkg-config.1.0
# compiler     4.02.1
# exit-code    127
# env-file     /Users/Henri/ocamlbrew/ocaml-4.02.3/.opam/4.02.1/build/conf-pkg-config.1.0/conf-pkg-config-79808-7afd23.env
# stdout-file  /Users/Henri/ocamlbrew/ocaml-4.02.3/.opam/4.02.1/build/conf-pkg-config.1.0/conf-pkg-config-79808-7afd23.out
# stderr-file  /Users/Henri/ocamlbrew/ocaml-4.02.3/.opam/4.02.1/build/conf-pkg-config.1.0/conf-pkg-config-79808-7afd23.err



=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
The following actions were aborted
  ∗  install conf-ncurses 1    
  ∗  install ocamlfind    1.6.1
The following actions failed
  ∗  install conf-pkg-config 1.0
No changes have been performed

=-=- conf-pkg-config.1.0 troobleshooting =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
=> This package relies on external (system) dependencies that may be missing. `opam depext conf-pkg-config.1.0' may help you find the correct installation for your
   system.
MacAir-H:~ Henri$ 
```

Running opam depext shows no dependency missing

Regards
Henri

--- a/packages/ocamlfind/ocamlfind.1.3.1/opam
+++ b/packages/ocamlfind/ocamlfind.1.3.1/opam
@@ -9,5 +9,4 @@ build: [
 available: [(ocaml-version >= "3.08") & (ocaml-version < "3.12.2")]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]

--- a/packages/ocamlfind/ocamlfind.1.3.2/opam
+++ b/packages/ocamlfind/ocamlfind.1.3.2/opam
@@ -9,5 +9,4 @@ build: [
 ocaml-version: [> "3.12.2"]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]

--- a/packages/ocamlfind/ocamlfind.1.3.3/opam
+++ b/packages/ocamlfind/ocamlfind.1.3.3/opam
@@ -10,5 +10,4 @@ build: [
 available: [(ocaml-version >= "3.08") & (ocaml-version <= "4.01.0")] # ocamlfind uses Arg.align of 3.08
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]

--- a/packages/ocamlfind/ocamlfind.1.4.0/opam
+++ b/packages/ocamlfind/ocamlfind.1.4.0/opam
@@ -8,6 +8,5 @@ build: [
 ]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]
 available: [ocaml-version >= "3.08"] # ocamlfind uses Arg.align

--- a/packages/ocamlfind/ocamlfind.1.4.1/opam
+++ b/packages/ocamlfind/ocamlfind.1.4.1/opam
@@ -8,6 +8,5 @@ build: [
 ]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]
 available: [ocaml-version >= "3.08"] # ocamlfind uses Arg.align

--- a/packages/ocamlfind/ocamlfind.1.5.1/opam
+++ b/packages/ocamlfind/ocamlfind.1.5.1/opam
@@ -13,7 +13,6 @@ remove: [
 ]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]
 # needs 3.10.0 for ocamldep -modules but for some reason, when using
 # 3.11 an ocamlfind test needs native dynamic linking which is only from 3.12.0

--- a/packages/ocamlfind/ocamlfind.1.5.2/opam
+++ b/packages/ocamlfind/ocamlfind.1.5.2/opam
@@ -13,7 +13,6 @@ remove: [
 ]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]
 # needs 3.10.0 for ocamldep -modules but for some reason, when using
 # 3.11 an ocamlfind test needs native dynamic linking which is only from 3.12.0

--- a/packages/ocamlfind/ocamlfind.1.5.3/opam
+++ b/packages/ocamlfind/ocamlfind.1.5.3/opam
@@ -13,7 +13,6 @@ remove: [
 ]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]
 # needs 3.10.0 for ocamldep -modules but for some reason, when using
 # 3.11 an ocamlfind test needs native dynamic linking which is only from 3.12.0

--- a/packages/ocamlfind/ocamlfind.1.5.4/opam
+++ b/packages/ocamlfind/ocamlfind.1.5.4/opam
@@ -14,6 +14,5 @@ remove: [
 ]
 depends: [
   "conf-m4"
-  "conf-ncurses"
 ]
 available: [ocaml-version >= "3.12.0"] # ocamlfind refers to cmxs

--- a/packages/ocamlfind/ocamlfind.1.5.5/opam
+++ b/packages/ocamlfind/ocamlfind.1.5.5/opam
@@ -18,6 +18,5 @@ remove: [
 ]
 depends: [
   "conf-m4" {build}
-  "conf-ncurses"
 ]
 available: [ocaml-version >= "3.12.0"] # ocamlfind refers to cmxs

--- a/packages/ocamlfind/ocamlfind.1.5.6/opam
+++ b/packages/ocamlfind/ocamlfind.1.5.6/opam
@@ -19,5 +19,4 @@ remove: [
 available: [ocaml-version >= "3.12.0"] # ocamlfind refers to cmxs
 depends: [
   "conf-m4" {build}
-  "conf-ncurses"
 ]

--- a/packages/ocamlfind/ocamlfind.1.6.1/opam
+++ b/packages/ocamlfind/ocamlfind.1.6.1/opam
@@ -19,5 +19,4 @@ remove: [
 available: [ocaml-version >= "3.12.0"] # ocamlfind refers to cmxs
 depends: [
   "conf-m4" {build}
-  "conf-ncurses"
 ]

--- a/packages/ocamlfind/ocamlfind.1.6.2/opam
+++ b/packages/ocamlfind/ocamlfind.1.6.2/opam
@@ -23,5 +23,4 @@ remove: [
 available: [ocaml-version >= "3.12.0"] # ocamlfind refers to cmxs
 depends: [
   "conf-m4" {build}
-  "conf-ncurses" {build}
 ]

